build/images/base-windows/Dockerfile

```diff
@@ -12,52 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG GO_VERSION
-ARG NANOSERVER_VERSION
-
-FROM mcr.microsoft.com/windows/servercore:${NANOSERVER_VERSION} as windows-utility-base
-
+FROM --platform=linux/amd64 alpine:3.14 as antrea-build-windows
```

Comment:
We avoid using alpine images in the project. There may be some references that have fallen through the cracks, but please use ubuntu:22.04 is possible.

Reply:
sure, will update.